simd.dd

```diff
@@ -57,9 +57,25 @@ import core.simd;
 ---
 
 	$(P These types and operations will be the ones defined for the architecture
-	the compiler is targetting. If a particular CPU family has varying
+	the compiler is targeting. If a particular CPU family has varying
 	support for vector types, an additional runtime check may be necessary.
-	The compiler does not emit runtime checks; those must be done by the programmer.
+	The compiler does not emit runtime checks; those must be done by the 
+	programmer.
+	)
+	
+	$(P It is possible to test if the architecture the compiler is targeting
+	supports SIMD types by using:)
+---
+version(D_SIMD)
```

Comment:
No problem.  I don't mind me if I get a bit emotional when I see dmd-specific information being put into the spec.  There should be a dmd page for this kind of content.

> I've been trying to add more details to areas where I had some difficulty (I probably spent a few hours trying to figure out why it wouldn't compile when I didn't use -m64).

That is a limitation of using dmd you've found there, other compilers have a wider range of SIMD support.